modules/mapper-extras/build.gradle

```diff
@@ -21,3 +21,6 @@ esplugin {
     description 'Adds advanced field mappers'
     classname 'org.elasticsearch.index.mapper.MapperExtrasPlugin'
 }
+test.configure {
+    systemProperty 'es.script.exception_for_missing_value', 'true'
```

Comment:
Maybe it would be better to set this for all tests (eg in BuildPlugin), and override it for one test to check the bwc behavior?